pkg/baseline/packageIndex.json

```diff
@@ -113,6 +113,12 @@
         "4.0.3.0": "4.4.0",
         "4.0.4.0": "4.5.0"
       }
+    },
```

Comment:
Not necessarily.  Baseline is a minimum version.  Since we update the versions of the references with every release the minimum won't come into the picture here.